presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1722,6 +1722,11 @@ VeloxQueryPlanConverterBase::toVeloxQueryPlan(
     auto hiveOutputTableHandle =
         std::dynamic_pointer_cast<protocol::HiveOutputTableHandle>(
             createHandle->handle.connectorHandle);
+    if (hiveOutputTableHandle->bucketProperty != nullptr) {
```

Comment:
perhaps, VELOX_USER_CHECK_NULL(hiveOutputTableHandle->bucketProperty, ...)

to make sure this becomes a user error